ios/Classes/Keys.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Keys.swift
+//  customer_io
+//
+//  Created by ShahrozAli on 11/11/22.
+//
+
+import Foundation
+
+struct Keys {
```

Comment:
This comment applies to both Android and iOS `Keys` objects. 

I see value in code like this in that we are containing strings into 1 place. However, I wonder if there is another idea for how to solve this problem? I think it sounds like a lot of work to maintain this file over time. 

We tried a similar idea in the RH iOS app and we found ourselves forgetting often to add constants in the file instead of having string values in the app. 

One idea is to move the code that uses these values into a util class that encapsulates the strings into 1 file:

```kotlin
// Before
// CustomerIOPlugin file
getUserAgent() {
  ...
  val sdkVersion = getValue(Keys.PackageConfig.version)

// After
// Code moved into SDKPackageUtil
class SDKPackageUtilImpl: SDKPackageUtil {
  val userAgent: Client {
    val sdkVersion = getValue("version")
  }
}
```

The code above isn't quite accurate but I hope you get the idea. Instead of having 1 file that contains a long list of random constants we can have the hard-coded strings encapsulated in 1 file that does the task that we need it to do. 

Seems more testable and maintainable to me in some ways.